cypress/support/fragments/invoices/settingsInvoices.js

```diff
@@ -0,0 +1,62 @@
+import { Button, TextField, NavListItem, MultiColumnListHeader, EditableListRow, MultiColumnListCell } from '../../../../interactors';
+import InteractorsTools from '../../utils/interactorsTools';
+
+const settings = {
+  approvals: 'Approvals',
+  adjustments: 'Adjustments',
+  batchGroups: 'Batch groups',
+  batchGroupConfiguration: 'Batch group configuration',
+  voucherNumber: 'Voucher number'
+};
+
+export default {
+  settings: {
+    approvals: 'Approvals',
+    adjustments: 'Adjustments',
+    batchGroups: 'Batch groups',
+    batchGroupConfiguration: 'Batch group configuration',
+    voucherNumber: 'Voucher number'
+  },
+
+  selectSetting: (setting) => {
+    cy.do(NavListItem(setting).click());
+    cy.expect(MultiColumnListHeader({ id: 'list-column-name' }).exists());
+  },
+
+  createNewBatchGroup: (batchGroup) => {
+    cy.expect(Button({ id: 'clickable-add-batch-groups' }).exists());
+    cy.do([
+      Button({ id: 'clickable-add-batch-groups' }).click(),
+      TextField({ placeholder: 'name' }).fillIn(batchGroup.name),
+      TextField({ placeholder: 'description' }).fillIn(batchGroup.description),
+      Button('Save').click()
+    ]);
+  },
+
+  editBatchGroup: (batchGroup, rowNumber = 0) => {
+    cy.do([
+      EditableListRow({ index: rowNumber })
+        .find(Button({ icon: 'edit' })).click(),
+      TextField({ placeholder: 'name' }).fillIn(batchGroup.name),
```

Comment:
you can create fillRequiredFields() wrapper and use it into create and update functions

Reply:
Updated